### DISCUSSION TITLE
bug on repeated execution of a multi-step ussd code

### DISCUSSION THREAD

**Discussion by brisvi** [romellfudi/VoIpUSSD#95] [Q&A]
I don't know if it's a bug but when I execute a ussd code in several steps by passing a sim slot in parameter all the sequence executes well for the first time but if I launch again the code it stops at the first step.

**Comment by romellfudi** [comment]
Hi @brisvi please use the [Rev26](https://github.com/romellfudi/VoIpUSSD/tree/Rev26)

**Comment by douala-777** [comment]
Me too .. I use the Rev26 even though the problem is still there . Maybe I'm missing something ?
